setup.py

```diff
@@ -16,7 +16,6 @@
         'Django >=2.0.13, < 2.1',
         'djangorestframework',
         'psycopg2-binary',
-        'django-prices',
```

Comment:
Why do we remove it from the setup file?

Reply:
This was because I had just put it manually into this file. you will notice that in the next commit, its installed(in the correct manner)